test/e2e/basics.go

```diff
@@ -519,3 +519,127 @@ func testIgnorePaths(client kubernetes.Interface) kubetest.TestSuite {
 		}.Run(t)
 	}
 }
+
+func testLegacyTokenFlag(client kubernetes.Interface) kubetest.TestSuite {
+	return func(t *testing.T) {
+		legacyCommand := `curl --connect-timeout 5 -v -s -k --fail -H "Authorization: Bearer $(cat /var/run/secrets/tokens/requestedtoken)" https://kube-rbac-proxy.default.svc.cluster.local:8443/metrics`
```

Comment:
`/var/run/secrets/tokens/requestedtoken` does not actually house the legacy tokens. The below code appears to be creating such tokens:

https://github.com/brancz/kube-rbac-proxy/blob/670377fd78875cfcfc9a1c42e0b086da8bb81043/test/kubetest/kubernetes.go#L524-L544

We may actually want to rename the directory from `/var/run/secrets/tokens` to something else, like `/var/run/projected/tokens`, I personally find it a bit confusing.

You'll have to follow https://kubernetes.io/docs/concepts/configuration/secret/#serviceaccount-token-secrets to create a secret along with your other manifests. Use a different directory than for the short-lived SA tokens so that we can distinguish these two in tests.